samples/ClearView.Android/MainActivity.cs

```diff
@@ -15,7 +17,7 @@ protected override void OnCreate (Bundle savedInstanceState)
 			SetContentView (Resource.Layout.Main);
 
 			LinearLayout layout = FindViewById<LinearLayout> (Resource.Id.Layout);
-			layout.AddView (new ClearView (ApplicationContext));
+			layout.AddView (new VulkanViewSample(ApplicationContext, new ClearSample()));
```

Comment:
Could you please keep the original indentation here and in other places? (the missing space)

Reply:
I do not see this style very often and it's hard to enforce it. Visual Studio removes those spaces every time you copy and paste or even type the semicolon at the end of a line.

Should I use spaces before every brace?

Reply:
We use coding style explained here http://www.mono-project.com/community/contributing/coding-guidelines/

I wonder whether this might help you in VS, https://blogs.msdn.microsoft.com/dotnet/2016/12/15/code-style-configuration-in-the-vs2017-rc-update/